VerilogCodingStyle.md

```diff
@@ -965,12 +965,14 @@ facilitate the creation of IP that may be re-used across many projects.
 
 The preferred naming convention for all immutable constants is to use `ALL_CAPS`, but there are times when the use of `UpperCamelCase` might be considered more natural.
 
+The naming convention for parameter and localparam allows for the use of `ALL_CAPS` or `UpperCamelCase`. The allowance for `ALL_CAPS` is to accommodate this ubiquitous style found in the ASIC industry, and by all third party IP vendors.
+
 | Constant Type | Style Preference | Conversation |
 | ---- | ---- | ---- |
 | \`define            | `ALL_CAPS`       | Truly constant |
-| module parameter    | `UpperCamelCase` | truly modifiable by instantiation, not constant |
-| derived localparam  | `UpperCamelCase` | while not modified directly, still tracks module parameter |
-| tuneable localparam | `UpperCamelCase` | while not expected to change upon final RTL version, is used by designer to explore the design space conveniently |
+| module parameter    | `ALL_CAPS` or `UpperCamelCase` | truly modifiable by instantiation, not constant |
```

Comment:
Please switch the order here since our code base is predominantly `UpperCamelCase`.

    `UpperCamelCase` or `ALL_CAPS`